screens/Request/RequestScreenController.ts

```diff
@@ -43,6 +47,14 @@ export function useRequestScreen({ navigation }: MainRouteProps) {
     };
   }, []);
 
+  useEffect(() => {
+    BluetoothStateManager.getState().then((bluetoothState) => {
+      if(bluetoothState === 'PoweredOn' && isBluetoothDenied) {
+        requestService.send(RequestEvents.SCREEN_FOCUS())
+      }
+    });
+  }, [isActive]);
```

Comment:
It does not work for me in Android 11 when I toggle Bluetooth from the drawer. The app is still active so this side-effect won't trigger again for this case. We may need to listen for an [AppState focus event](https://docs.expo.dev/versions/latest/react-native/appstate/#focus---android).